### ISSUE TITLE
Proxy broken in the latest beta release

### ISSUE THREAD

**Issue by mihaialexe** [meetfranz/franz#1181]
**Describe the bug**
I've updated recently Franz to the latest release (5.0.0-beta22) and noticed that it messed up the proxy settings.
To be more precise, previously, when setting "use proxy" to a service, without filling the fields, i was able to make the service use "direct connection", bypassing the default proxy of the OS. Not disclosing the reason behind it, though 😬
Right now, doing that doesn't change anything. It would be nice to be fixed, since it's the only reason i'm paying for premium supporter account

**To Reproduce**
You will need to have a proxy configured. Preferably one that filters certain services (i.e. Whatsapp, or Messenger.com)

1. Go to 'Settings'
2. Click on 'Your services'
3. Select any of them and choose "use proxy" (without filling in the fields)
4. Check if the default OS proxy is overridden

**Expected behavior**
OS proxy is bypassed and uses direct connection

**Desktop:**
 - OS: Windows 10 b1083
 - Franz Version 5.0.0-b22

**Comment by adlk** [comment]
For bypassing proxy you need to use `direct://` as a proxy host. I'll add an option in the future that explicitly says "Bypass proxy".

The default setting now is to use the system proxy by default.

**Comment by mihaialexe** [comment]
Thanks for the quick reply. Just tested it and it works.
I filed a bug report, because previously it was working without `direct://` set as proxy

**Comment by adlk** [comment]
Yep, I'm aware. This was the default setting that broke Franz in a few corporate environments (whops)
But using the system proxy by default isn't usually the worst idea ;)
